docs/application/dotnet/guides/nui/resources.md

```diff
@@ -5,12 +5,46 @@ This tutorial describes the use of Image resources in NUI.
 
 ## Overview
```

Comment:
I would suggest writing a brief overview and the current information can follow up.

Reply:
@nawab-samsung: maybe I will change the title from Overview to Examples or Usage ? Short description is just above that paragraph so I don't want to double information. What do you think about that?

Reply:
@mihashco probably that will be good since there exists the overview (one line though)
So we can change the heading 

## Overview

to  

## Image handling methods  

- That's my suggestion, feel free to change as per your liking if this is not suitable.